processes/processes.md

```diff
@@ -156,7 +156,7 @@ Response
   "alertEmailAddress": "",
   "alertSMS": "",
   "description": "Description here",
-  "lastModifiedBy": "qaseed.brendan.lui",
+  "lastModifiedBy": "usaseed.john.adams",
```

Comment:
@hal9000-hub the name does not have to contain 'seed' in it. Can be just `usa.john.adams`.

Reply:
yep it can just be usa.john.adams

Do you want me to create a new pull request?